### ISSUE TITLE
Change Traffic Monitor 2.0 Manager Parameters To Be More Readable

### ISSUE THREAD

**Issue by robert-butts** [Comcast/traffic_control#2012]
Right now, the managers take a lot of parameters. This is hard to read, and adding or removing any makes a hard-to-read diff of the whole line.

These should be made more manageable somehow.

**Comment by robert-butts** [comment]
I've tried to make structs several times, and not liked the result each time. 

Structs for each manager make the function definition a little easier to read, but make it twice as verbose to pass the params (constructing the param struct by-name), and make accessing the params unnecessarily verbose as well (e.g. `params.CombinedStates`).

I've also tried making a "world" struct, with all the data all managers need. The problem with that, is that each manager creates some new data. Then, the "world" object is only partially-initialized when passed to all but the last manager. Thus, someone who doesn't fully and completely understand the managers could access an uninitialized member.

For now, I'll probably put each param on its own line, which will be more vertically verbose, but it'll make it easier to read, and make diffs better.
